cmd/newallocation.go

```diff
@@ -258,6 +274,8 @@ var newallocationCmd = &cobra.Command{
 				ThirdPartyExtendable: thirdPartyExtendable,
 				Force:                force,
 				IsEnterprise:         isEnterprise,
+				StorageVersion:       storageVersion,
+				ManagingWallet:       managingWallet,
```

Comment:
There is no option in CreateAllocationOptions for Managing wallet.

Reply:
Removed. Got confused bw Delegate and Managing wallet